pkg/minishift/provisioner/minishift_detector_test.go

```diff
@@ -71,7 +71,7 @@ VARIANT_VERSION="1.0.0-alpha.1"
 
 	osRelease, _ := provisioner.GetOsReleaseInfo()
 	if osRelease.Variant != "minishift" {
-		t.Fatal("Release info should contain 'minishift' variant")
+		t.Fatal("Release info must contain 'minishift' variant")
```

Comment:
Actually in the context of this test, "should" is better. I setup a test harness and had an expectation that something happened. Then I executed the test and my expectation was not met. In this case I think it makes sense to say: "It should have contained". It should be there, but for some reason I don't know it was not.